addons/l10n_de_skr03/migrations/13.0.3.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,41 @@
+---Models in module 'l10n_de_skr03'---
+---Fields in module 'l10n_de_skr03'---
+---XML records in module 'l10n_de_skr03'---
+NEW account.account.template: l10n_de_skr03.account_1411
+account.chart.template: l10n_de_skr03.l10n_de_chart_template (noupdate switched)
```

Comment:
something to do here

Reply:
Done.
I added a pre-migration script.
Thank you.